hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/volume/TestReservedVolumeSpace.java

```diff
@@ -95,15 +95,18 @@ public void testVolumeCapacityAfterReserve() throws Exception {
         HDDS_DATANODE_DIR_DU_RESERVED_PERCENT_DEFAULT);
 
     long volumeCapacity = hddsVolume.getCurrentUsage().getCapacity();
+    long volumeAvailable = hddsVolume.getCurrentUsage().getAvailable();
```

Comment:
Please get usage once:

```java
    SpaceUsageSource currentUsage = hddsVolume.getCurrentUsage();
    long volumeCapacity = currentUsage.getCapacity();
    long volumeAvailable = currentUsage.getAvailable();
```

(needs import, so not adding as github suggestion)

Reply:
Updated, PTAL.